recipes/eip712/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "eip712" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "3997dace7e581b66a84d106a10baac47a3f6c94095d79c7d0971ca0ede1926ad" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8.0
+    - hexbytes
+    - setuptools
+    - setuptools-scm
+    - wheel
+    - pip
+  run:
+    - python >=3.8.0
+    - dataclassy >=0.8.2,<1
+    - eth-abi >=4.1.0,<5
+    - eth-account >=0.8.0,<0.9
+    - eth-hash
+    - eth-typing >=3.3.0,<4
+    - eth-utils >=2.1.0,<3
+    - hexbytes >=0.3.0,<1
+
+test:
+  imports:
+    - eip712
+  commands:
```

Comment:
As just mentioned in the other review, you should be switching to the release tarball from GitHub instead of PyPI for the sources. This will then also include the test folder but it also much more reliable than a manual git clone in the tests.